pkg/config/system_probe.go

```diff
@@ -171,7 +171,9 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(smNS, "enable_go_tls_support"), false)
 
 	cfg.BindEnvAndSetDefault(join(smNS, "enable_java_tls_support"), false)
```

Comment:
shouldn't we change that as well?

Reply:
could we change `enable_go_tls_support` as well ?
I mean it would nice to be consistent, WDYT ?

Reply:
I'm in favor of @brycekahle original comment https://github.com/DataDog/datadog-agent/pull/15601#discussion_r1106116893